packages/fast-components-class-name-contracts-msft/src/flipper/index.ts

```diff
@@ -0,0 +1,10 @@
+import { IButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * The class name contract for the flipper component
+ */
+export interface IFlipperClassNameContract extends IButtonClassNameContract {
```

Comment:
Wondering the purpose of extending button class name contact? Is the idea that flipper is a special button so we should tie them together?

Reply:
Flipper is built out of the button base component and specific to MSFT so, yes we are extending  upon base button here.